### PR TITLE
fix(prob volumes)

### LINKED ISSUE #62: Make Volume data classification labels correspond with new notebooks
At the moment protected B notebooks cannot launch with volumes because the labels are not set correctly in the UI. We just need new volumes associated to ProB notebooks to come with a ProB label.

### PR DESCRIPTION
**Pull request by Jose-Matsuda**
Add a label indicating that the volume is protected b
iff notebook.Labels[notebook.statcan.gc.ca/protected-b] exists

Closes #62

### CODE REVIEW

File: notebooks.go [hunk 1]
```diff
@@ -478,14 +496,14 @@ func (s *server) NewNotebook(w http.ResponseWriter, r *http.Request) {
 	//Add Language
 	//Validate that the language format is valid (language[_territory])
 	match, err := regexp.MatchString("^[[:alpha:]]{2}(_[[:alpha:]]{2})?$", req.Language)
-		if (err != nil || !match) {
-			var errLanguageFormat = errors.New("Error: the value of KF_LANG environment variable ('" + req.Language + "') is not a valid format (e.g 'en', 'en_US', ...)")
-			s.error(w, r, errLanguageFormat)
-			return
-		}
+	if err != nil || !match {
+		var errLanguageFormat = errors.New("Error: the value of KF_LANG environment variable ('" + req.Language + "') is not a valid format (e.g 'en', 'en_US', ...)")
```
Comment (wg102) [inline notebooks.go:500]: Is there a way to just send a message back to front end and have a i18n message?

File: notebooks.go [hunk 2]
```diff
@@ -232,23 +232,41 @@ func (s *server) GetNotebooks(w http.ResponseWriter, r *http.Request) {
...
-				Resources: corev1.ResourceRequirements{
-					Requests: corev1.ResourceList{
-						corev1.ResourceStorage: req.Size,
+		if _, ok := notebook.Labels["notebook.statcan.gc.ca/protected-b"]; ok {
+			pvc = corev1.PersistentVolumeClaim{
+				ObjectMeta: v1.ObjectMeta{
+					Name:      req.Name,
+					Namespace: notebook.Namespace,
+					Labels:    map[string]string{"data.statcan.gc.ca/classification": "protected-b"},
+				},
```
Comment (wg102) [inline notebooks.go:243]: Is there a way in go to just have this part conditional?

**Review by wg102 — CHANGES_REQUESTED** [review]

File: notebooks.go (hunk 1 above)
Comment (Jose-Matsuda) [reply notebooks.go:500]: This wasn't a part of our changes it just formatted I think

File: notebooks.go (hunk 2 above)
Comment (Jose-Matsuda) [reply notebooks.go:243]: I don't think so?

**Pull request merged**